sdk/browser/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# release 7.9.0 (2022-12-09)
+fix: filter corrupted credentials from getCredentials()
```

Comment:
Please follow the previous approach (e.g. added/updated/fixed), I agree it is discussable to have commit message, existing approach or something else, but if everyone use his own way to tell about changes - it leads to mess

Reply:
ok